examples/tutorials/DeepChem_AntibodyTutorial_Simplified.ipynb

```diff
@@ -0,0 +1,3696 @@
+{
```

Comment:
<p>This is probably worth splitting into its own tutorial and expanding with antibody property prediction models for a full design loop</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4012/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>